TestsProject/app/autoload.php

```diff
@@ -1,4 +1,6 @@
 <?php
 
 $loader = require(__DIR__.'/../../vendor/autoload.php');
-$loader->add('', __DIR__.'/../src');
\ No newline at end of file
+$loader->add('', __DIR__.'/../src');
+
+require_once __DIR__.'/AppKernel.php';
```

Comment:
Please could you add a newline at the end of this file to [keep git happy](https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file)?